__e2e__/commands/showcommands.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const path = require('path');
+const execa = require('execa');
+const test = require('ava');
+const fs = require('fs');
+
+const rootCommand = path.join(process.cwd(), 'bin/index.js');
+const configFilePath = path.join(process.cwd(), 'config.json');
+
+const validConfigJson = {
+  userName: 'validConfigJson',
+  taskCount: 0,
+};
+
+test.serial('no config file for showcommands', async t => {
+  if (fs.existsSync(configFilePath)) await execa('rm', ['config.json']);
+  const { stdout } = await execa(rootCommand, ['showcommands'], {
+    reject: false,
+  });
+  t.snapshot(stdout);
+});
+
+test.serial('config file present for showcommands', async t => {
+  fs.writeFileSync('config.json', JSON.stringify(validConfigJson));
+  const { stdout } = await execa(rootCommand, ['showcommands'], {
+    reject: false,
+  });
+  t.snapshot(stdout);
+  await execa('rm', ['config.json']);
```

Comment:
Not a cross-platform solution. Go with [rimraf](https://github.com/isaacs/rimraf) instead.

Reply:
Is it fine to go with [fs unlink sync](https://nodejs.org/api/fs.html#fs_fs_unlinksync_path)? It seems better for promises

Reply:
Yeah, let's not introduce new dependencies.

Reply:
Not sure if I should use [fs unlink](https://nodejs.org/api/fs.html#fs_fs_unlink_path_callback) instead. The tests are passing with `fs.unlinkSync` though.